test/integration/openid4vp_test.go

```diff
@@ -24,61 +25,82 @@ import (
 )
 
 func TestOpenID4VPFullFlow(t *testing.T) {
-	testHelper := newVPTestHelper(t)
+	type test struct {
+		issuerProfileID   string
+		walletDIDMethod   string
+		verifierProfileID string
+		verifierDIDMethod string
+	}
 
-	issuedCredentials := testHelper.issueCredentials(t)
+	tests := []test{
+		{
+			issuerProfileID:   "bank_issuer",
+			walletDIDMethod:   "ion",
+			verifierProfileID: "v_myprofile_jwt",
+			verifierDIDMethod: "ion",
+		},
+	}
 
-	setup := oidc4vp.NewSetup(testenv.NewHttpRequest())
+	for _, tc := range tests {
+		fmt.Println(fmt.Sprintf("running tests with : issuerProfileID=%s walletDIDMethod=%s verifierDIDMethod=%s",
```

Comment:
Same formatting issue with the `:`

Reply:
https://github.com/trustbloc/wallet-sdk/pull/140#discussion_r1061836754